Foundation/NSLog.swift

```diff
@@ -41,3 +41,11 @@ public func NSLog(_ format: String, _ args: CVarArg...) {
         NSLogv(format, $0) 
     }
 }
+
+// hook for System.out.println()
+#if os(Android)
+public var ANLog = {
```

Comment:
My suggestion is to in the backing for NSLog use android_log_print and similarly for the print function. 

Technically you could `funopen` a new descriptor for stdout to pipe logging on Android (I have done that on previous projects that I have worked on and it worked reasonably well for logging)